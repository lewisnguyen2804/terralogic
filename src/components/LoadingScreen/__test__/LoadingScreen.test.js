import React from 'react'
import { shallow } from 'enzyme'
import LoadingScreen from '../';

describe('Test LoadingScreen', () => {
    it('Test what?', () => {
        const loadingScreen = shallow(<LoadingScreen  />);
        expect(loadingScreen).toMatchSnapshot()
    });
});