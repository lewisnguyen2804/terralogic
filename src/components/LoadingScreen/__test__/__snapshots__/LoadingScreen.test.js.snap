// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test LoadingScreen Test what? 1`] = `ShallowWrapper {}`;
